src/components/collapsible/CollapsibleReducers.ts

```diff
@@ -0,0 +1,79 @@
+import * as _ from 'underscore';
+import {IReduxActionsPayload} from '../../ReactVapor';
+import {IReduxAction} from '../../utils/ReduxUtils';
+import {CollapsibleActions} from './CollapsibleActions';
+
+export interface CollapsibleState {
+    id: string;
+    expanded: boolean;
+}
+
+/**
+ * @deprecated use CollapsibleState instead
+ */
+export interface ICollapsibleContainerState extends CollapsibleState {}
+
+export const collapsibleInitialState: CollapsibleState = {
+    id: undefined,
+    expanded: undefined,
+};
+
+/**
+ * @deprecated use collapsibleInitialState instead
+ */

```

Comment:
Other than in tests, I'm not sure someone would actually need to import this.  I think this is safe to remove.

Reply:
Same with `collapsibleContainersInitialState`